src/internal/app/app.go

```diff
@@ -131,6 +131,10 @@ func (a *application) Initialize() error {
 		return errors.Wrap(err, "failed to initialize things")
 	}
 
+	if err := a.cfgService.Save(); err != nil {
```

Comment:
Why do we need to call `Save` here?

Reply:
It's just a way to ensure that configs are up-to-date when the app is initialised. 
Right now, if the new version has a new field the app will use the old config file (without a field) until the app flow updates the config file.

Reply:
New fields are reflected in a filesystem after a config update, that's correct, but the important thing is that they are reflected in memory right after an application startup. IMO it's optional, but we can keep it.